configurator_test.go

```diff
@@ -62,14 +104,22 @@ func (c *config) Update(_ uint64, data interface{}) {
 		}
 
 	case []*api.ServiceEntry:
-		// nothing here yet
+		c.service = data.([]*api.ServiceEntry)
+		c.t.Logf("Update: I have %d services in my list", len(c.service))
 
 	default:
 		c.t.Log("Typecast failed")
 		c.t.Fail()
 	}
 }
 
+func (c *config) copy() *config {
+	c.RLock()
+	defer c.RUnlock()
+	cc := *c
```

Comment:
This is failing govet in the travis build: https://travis-ci.org/myENA/consultant/jobs/236450500

```
configurator_test.go:119: assignment copies lock value to cc: consultant_test.config
```